.github/workflows/ci.yml

```diff
@@ -673,6 +673,13 @@ jobs:
         run: gcloud storage cp "${TF_BINARIES_GCS_ARTIFACT}" "${TF_BINARIES_ARCHIVE}"
       - name: "Extracting TF binaries archive"
         run: tar -xf "${TF_BINARIES_ARCHIVE}"
+      - name: "DEBUG: see what's in the binaries archive"
+        run: find $(dirname ${TF_BINARIES_ARCHIVE}) -maxdepth 2
```

Comment:
This is rough around the edges, but I was just able to reproduce what the CI does locally with this sequence of commands: https://gist.github.com/ScottTodd/bcb137207c7320865442f024e61c61d2

The trickiest details are in how Docker handles mounted directories and how artifacts are copied between jobs

Reply:
Re: TF_BINARIES_OUTPUT_DIR, what commit or branch did you run this on?  The new tool doesn't end up in TF_BINARIES_OUTPUT_DIR, since we're no longer building with bazel / copying around symlinks.  It's installed when the package is pip installed (although, as you can see from my commits, the environment still isn't aware of the installed commands).

Reply:
I need to disable 5 tests in `generated_e2e_test_iree_artifacts.cmake`, but I assume it's not a good idea to modify that file directly.  I've tried to traverse the numerous layers involved but haven't figured out the core source of truth yet.  Going to continue on it tomorrow.

Reply:
instructions: https://github.com/openxla/iree/tree/main/tests/e2e/test_artifacts#updating-test-artifacts (and maybe @pzread or someone else could help you with the specific targets)

Reply:
Is something going on with github CI right now? I can't get anything but the CLA agreement test to run.  Only 7 other jobs are showing up, and they all say `Expected — Waiting for status to be reported`.

Reply:
Should this step have been removed before merging?